docs/sources/reference/components/otelcol.extension.jaeger_remote_sampling.md

```diff
@@ -115,7 +115,7 @@ Name                     | Type      | Description
 -------------------------|-----------|----------------------------------------------------------------------------|-------------------|---------
 `endpoint`               | `string`  | `host:port` to listen for traffic on.                                      | `"0.0.0.0:14250"` | no
 `transport`              | `string`  | Transport to use for the gRPC server.                                      | `"tcp"`           | no
-`max_recv_msg_size`      | `string`  | Maximum size of messages the server will accept. 0 disables a limit.       |                   | no
+`max_recv_msg_size`      | `string`  | Maximum size of messages the server will accept.                           | `"4194304KiB"`    | no
```

Comment:
So I don't think this value is quite correct. gRPC says that the default value is 4MB, but 4194303KiB is 4GiB.

Meanwhile, there's a strange behavior in the units library we use where `4MB` would get interpreted as `4MiB`, which is incorrect. I'm going to work on a PR to fix this behavior since I'd consider it a fairly subtle bug. 

In the meantime, I'm not sure what value we should document here. The most correct thing would be `"4000000"` to match gRPCs defaults, though I'm not sure that's extremely easy for users to understand. `"4MB"` would be the easiest to reason about, but you're not able to explicitly configure `"4MB"` and have it actually be 4MB (it'll be 4MiB instead).

cc @clayton-cornell What would be the best thing to do for the documentation here? 

(This applies for all instances of this change in the other files too)

Reply:
Sorry, I'm wrong about a few things here.

https://github.com/grpc/grpc-go/blob/1e8b9b7fc655b756544cf10fded8f0d2fa3226e8/server.go#L56 

This is the default value; which is 4MiB, not 4MB as they documented elsewhere. So the correct string here would be `"4MiB"`

Reply:
We do have distinctions between MiB and MB in other places/products.  So... `4MiB` sounds reasonable to me from a doc standpoint.

Reply:
I fixed it to `4MiB` in HEAD commit.